Makefile

```diff
@@ -1,4 +1,4 @@
-CFLAGS  = -O3 -march=native -ggdb3 -m32 -std=gnu99 -fshort-wchar -Wno-multichar -Iinclude -mstackrealign
+CFLAGS  = -O3 -ggdb3 -m32 -std=gnu99 -fshort-wchar -Wno-multichar -Iinclude -mstackrealign
```

Comment:
I think this is a good idea, but probably need to remove it from the other Makefiles as well, I should clean up the build :)